src/openrct2/Editor.cpp

```diff
@@ -87,7 +87,7 @@ namespace Editor
         gS6Info.category = SCENARIO_CATEGORY_OTHER;
         viewport_init_all();
         rct_window* mainWindow = context_open_window_view(WV_EDITOR_MAIN);
-        mainWindow->SetLocation(2400, 2400, 112);
+        mainWindow->SetLocation({ 2400, 2400, 112 });
```

Comment:
```suggestion
        mainWindow->SetLocation(TileCoordsXYZ{ 75, 75, 14 }.ToCoordsXYZ());
```

Same for the other calls.